src/background/adblocker.js

```diff
@@ -224,6 +296,9 @@ function injectStyles(styles, tabId, frameId) {
 }
 
 async function injectCosmetics(details, config) {
+  const isBootstrap = config.bootstrap;
+  const scriptletsOnly = Boolean(config.scriptletsOnly);
```

Comment:
We are in control over the config, and `undefined` is just fine, you don't need to convert it to Boolean.